### PR TITLE
Using design system breakpoint values across ui/

### PR DESCRIPTION
**Pull request by ryanml**
Makes use of the breakpoint constants from the design system for code consistency (no changes to layout/functionality)

### CODE REVIEW

**Review by mcmire — DISMISSED** [review]
This LGTM!

File: ui/components/app/modals/deposit-ether-modal/index.scss [hunk 1]
```diff
@@ -109,7 +109,7 @@
       padding-bottom: 20px;
       align-self: flex-start;
 
-      @media screen and (min-width: 575px) {
+      @media screen and (min-width: $break-small) {
```
Comment (adonesky1) [inline ui/components/app/modals/deposit-ether-modal/index.scss:112]: Is this one and the one right below it correct? Shouldn't it be `min-width: $break-large` or `max-width: $break-small`?

File: ui/components/app/multiple-notifications/index.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
   right: 0;
   margin: 8px;
 
-  @media screen and (max-width: 576px) {
+  @media screen and (max-width: $break-large) {
```
Comment (adonesky1) [inline ui/components/app/multiple-notifications/index.scss:11]: same here?

File: ui/pages/settings/contact-list-tab/index.scss [hunk 3]
```diff
@@ -7,7 +7,7 @@
 .address-book {
   flex: 0.4 1 100%;
 
-  @media screen and (max-width: 576px) {
+  @media screen and (max-width: $break-large) {
```
Comment (adonesky1) [inline ui/pages/settings/contact-list-tab/index.scss:10]: Same here

File: ui/pages/settings/contact-list-tab/index.scss [hunk 4]
```diff
@@ -255,7 +255,7 @@
     width: auto;
     border-radius: 100px;
 
-    @media screen and (max-width: 576px) {
+    @media screen and (max-width: $break-large) {
```
Comment (adonesky1) [inline ui/pages/settings/contact-list-tab/index.scss:258]: and here

File: ui/pages/settings/contact-list-tab/index.scss [hunk 5]
```diff
@@ -269,7 +269,7 @@
 .address-book-contact-content {
   flex: 0.4 1 100%;
 
-  @media screen and (max-width: 576px) {
+  @media screen and (max-width: $break-large) {
```
Comment (adonesky1) [inline ui/pages/settings/contact-list-tab/index.scss:272]: and here

**Comment by ryanml** [comment]
@adonesky1 thanks for finding some of these existing fall throughs in our responsive breakpoints! I'll go through these, and audit and fix them 👍

**Comment by brad-decker** [comment]
@ryanml did we get design team approval on these break points being official? I can't remember if these were firm or not when I moved them to the design-system folder. 😞 Thank you so much for doing this though!

File: ui/components/app/modals/deposit-ether-modal/index.scss (hunk 1 above)
Comment (ryanml) [reply ui/components/app/modals/deposit-ether-modal/index.scss]: Fixed

File: ui/components/app/multiple-notifications/index.scss (hunk 2 above)
Comment (ryanml) [reply ui/components/app/multiple-notifications/index.scss]: Fixed

File: ui/pages/settings/contact-list-tab/index.scss (hunk 3 above)
Comment (ryanml) [reply ui/pages/settings/contact-list-tab/index.scss]: Fixed

File: ui/pages/settings/contact-list-tab/index.scss (hunk 4 above)
Comment (ryanml) [reply ui/pages/settings/contact-list-tab/index.scss]: Fixed

File: ui/pages/settings/contact-list-tab/index.scss (hunk 5 above)
Comment (ryanml) [reply ui/pages/settings/contact-list-tab/index.scss]: Fixed

**Comment by ryanml** [comment]
Instances of breakpoint fall-throughs highlighted by @adonesky1 plus other ones found have been fixed. cc: @mcmire @brad-decker

**Review by darkwing — APPROVED** [review]

**Review by mcmire — COMMENTED** [review]
I have a question. Typically I've seen either `min-width` or `max-width` used in stylesheets, but not both. Can we assume that the base size is the smallest size and fold `max-width` into its parent level?

**Comment by ryanml** [comment]
@mcmire - you're right that it's typically one or the other (with a base rule setting the precedence for largest or smallest)
I'm hesitant, in a single PR, to do the larger piece of refactoring in order to implement what you mentioned. I'm happy to do it in future PR's, in groups of similar components at once. I think this PR could possibly provide a good basis from which to kick that off.

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Pull request merged**